filesys/lvm2/BUILD

```diff
@@ -1,24 +1,47 @@
+cp -a ../LVM2.$VERSION ../LVM2-initramfs &&
+
 if ! [[ $(arch) == x86_64 ]]; then
   CFLAGS+=" -fPIE " &&
   LDFLAGS+=" -pie "
 fi
 
+OPTS+=" --with-udev_prefix=/usr"
 if module_installed systemd; then
-  OPTS+=" --with-udev_prefix=/usr \
-          --with-systemdsystemunitdir=$(pkg-config systemd --variable=systemdsystemunitdir)"
+  OPTS+=" --with-systemdsystemunitdir=/usr/lib/systemd/system"
+fi
+
+if ( module_installed udev ) || ( module_installed eudev ); then
```

Comment:
You don't need parens here.  The parens just cause excess forks to happen.

    if module_installed udev || module_installed eudev; then

would be better.